examples/web/README.md

```diff
@@ -104,9 +111,9 @@ configuration to the correct notification service:
 
 ![Alertmanager alerts dashboard showing the alerts firing](../../assets/alertmanager-alert-example.png)
 
-This demo example has a [minimal configuration](./alertmanager.yml) for alerts
+This demo example has a [minimal configuration](./configs/alertmanager.yml) for alerts
 that expects a file `slack_url.txt` to be passed in docker-compose context.
```

Comment:
How does it know the auth details? Is that included in the slack URL?

Reply:
The slack URL is a secret that has the auth to post in channels yes